apps/barista-design-system/src/pages/overview-page/components/component-preview-tile/component-preview-tile.scss

```diff
@@ -0,0 +1,137 @@
+@import '../../../../../libs/barista-components/core/src/style/colors';
+@import '../../../../styles/global/variables';
+
+$tilesize: 290px;
+$tilefontsize: 28px;
+$tilefontsize-big: 42px;
+$gridgap: 20px;
+$imagesize: 156px;
+
+@mixin tile-theme($link-color, $hover-color, $active-color) {
+  color: $link-color;
+
+  :hover,
+  &:hover {
+    border-color: $hover-color;
+    text-decoration: none;
+
+    &.ba-tile-image-preview {
+      border-bottom: 1px solid #ddd;
+    }
+  }
+
+  :focus,
+  &:focus {
+    outline: 2px solid $hover-color;
+  }
+
+  :active,
+  &:active {
+    border-color: $active-color;
+  }
+
+  .ba-tile-image {
+    color: $hover-color;
+  }
+
+  .ba-tile-badge-favorite,
+  &.ba-tile-badge-favorite {
+    .ba-tile-status-icon-triangle {
+      fill: $link-color;
+    }
+  }
+
+  .ba-tile-badge-workinprogress,
+  &.ba-tile-badge-workinprogress {
+    .ba-tile-status-icon-triangle {
+      fill: #ffffff;
+    }
+
+    img {
+      transform: rotate(90deg);
+    }
+  }
+}
+
+:host {
+  display: block;
+  background-color: #ffffff;
+  border: 1px solid $gray-200;
+  border-radius: 3px;
+  height: 100%;
+  min-width: $tilesize;
+  position: relative;
+  overflow: hidden;
+
+  @include tile-theme($turquoise-600, $turquoise-600, $turquoise-700);
+
+  .ba-tile-badge-warning {
+    .ba-tile-status-icon-triangle {
+      fill: $yellow-600;
+    }
+  }
+
+  .ba-tile-content {
+    display: flex;
```

Comment:
why do you need `display: flex` here? with `height: auto` it ends up being exactly the same as if you kept it as a block element. you could just remove all the flex stuff for the content.